src/plugins/discover/public/application/components/sidebar/discover_field.tsx

```diff
@@ -289,13 +301,13 @@ export function DiscoverField({
   const isRuntimeField = Boolean(indexPattern.getFieldByName(field.name)?.runtimeField);
   const isUnknownField = field.type === 'unknown' || field.type === 'unknown_selected';
   const canEditField = onEditField && (!isUnknownField || isRuntimeField);
-  const displayNameGrow = canEditField ? 9 : 10;
+  const canDeleteField = onDeleteField && isRuntimeField;
   const popoverTitle = (
     <EuiPopoverTitle style={{ textTransform: 'none' }} className="eui-textBreakWord">
-      <EuiFlexGroup responsive={false}>
-        <EuiFlexItem grow={displayNameGrow}>{field.displayName}</EuiFlexItem>
+      <EuiFlexGroup responsive={false} gutterSize="xs">
+        <EuiFlexItem grow={true}>{field.displayName}</EuiFlexItem>
```

Comment:
Consider wrapping `{field.displayName}` in a heading element. For Lens, an `h5` made the most sense, but you may want a different level here, depending on your existing heading hierarchy.